aries-site/src/examples/templates/feedback/PowerOnExample.js

```diff
@@ -0,0 +1,205 @@
+    <Layer>
+      <Box
+        fill="vertical"
+        overflow="auto"
+        width={!['xsmall', 'small'].includes(breakpoint) ? 'medium' : undefined}
+        pad="medium"
+        gap="medium"
+      >
+        <Heading level={4} size="small" margin="none">
+          Feedback on powering devices
+        </Heading>
+        {children}
+        <Footer direction="row" justify="end" gap="small">
+          <Button onClick={onClose} label="Cancel" />
+          <Button
+            onClick={onClick}
+            label="Submit Feedback"
+            primary
+            type="submit"
+          />
+        </Footer>
+      </Box>
+    </Layer>
+  );
+};
+
+export const PowerOnExample = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isSuccessful, setIsSuccessful] = useState(false);
+  const [rating, setRating] = useState();
+  const [showFeedback, setShowFeedback] = useState(false);
+
+  useEffect(() => {
+    setTimeout(() => {
+      if (isSuccessful) {
+        setShowFeedback(true);
+      }
+    }, 1000);
+  }, [isSuccessful]);
+
+  // simulating a call to a power on API call. Demo purposes
+  const PowerOnLoading = () => {
+    setIsLoading(true);
+    setTimeout(() => {
+      setIsLoading(false);
+      setIsSuccessful(true);
+    }, 3000);
+  };
+
+  // fetch call to POST api should be in onSubmit
+  // for demo purposes disabling actual API call
+  // submitFeedback();
+  const onSubmit = useCallback(value => {
+    const data = {
+      values: {
+        // url for the route which the feedback survey is presented
+        fullURL: url,
+        // type of device. Possible values 'desktop', 'tablet', 'mobile'.
+        deviceType: deviceType,
+        // QIDs are unique to each feedback instance and will be provided when
+        // coordinating with program manager, Doris Singer (doris.singer@hpe.com).
+        QID1: value.value['power-rating-example'],
+        QID2_TEXT: value.value['power-textArea-example'],
+      },
+    };
+    // dev uses POST https://api.qualtrics.com/{survey-id}-create-a-new-response
+    fetch(API_URL, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+        'X-API-TOKEN': API_TOKEN,
+      },
+      body: JSON.stringify(data),
+    })
+      .then(response => response.json())
+      .catch(error => {
+        // Error handling here
+      });
+  }, []);
+
+  return (
+    <Card width="large" gap="medium" justify="center" pad="medium">
+      <Box>
+        <Heading level={1} size="small" margin="none">
+          Power On
+        </Heading>
+        <Anchor href="#" label="Learn more about server actions" />
+      </Box>
+      {!isSuccessful ? (
+        <Text size="large">You are powering on 1 device</Text>
+      ) : (
+        <Box align="center" gap="xsmall" direction="row">
+          <StatusGoodSmall color="green" />
+          <Text size="large">Power On initiated for 1 device.</Text>
```

Comment:
This step should have a button for the user to close/complete the flow. onClick will then either close the modal dialog or show the survey within the same modal dialog depending on "show survey logic."

"Show survey logic" --> in the real world, we don't want to show the survey every time a flow is completed. There is likely some set of rules we will want set up so that the survey is only shown to a subset of users, that a single user only sees the survey once per (month, quarter, etc.).